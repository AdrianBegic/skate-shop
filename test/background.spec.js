import { mount } from '@vue/test-utils';
import BackGround from '../components/background.vue'


describe('BackGround component', () => {
    describe('Default mounting properties', () => {
      it('should mount successfully', () => {
        const wrapper = mount(BackGround);
  
        expect(wrapper.vm._isMounted).toBeTruthy();
  
        expect(wrapper).toMatchSnapshot();
      });
  
      it('should unmount successfully', () => {
        const wrapper = mount(BackGround);
  
        const BackGroundInstance = wrapper.vm;
        BackGroundInstance.$destroy();
        expect(BackGroundInstance._isDestroyed).toBeTruthy();
  
        expect(wrapper).toMatchSnapshot();
      });
    })})