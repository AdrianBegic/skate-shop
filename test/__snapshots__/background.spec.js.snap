// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackGround component Default mounting properties should mount successfully 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;

exports[`BackGround component Default mounting properties should unmount successfully 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
